src/nft-card.ts

```diff
@@ -50,123 +71,101 @@ export class NftCard extends LitElement {
         transform: rotateY(180deg);
       }
     `
-  }
-
-  /* User configurable properties */
-  @property({type: Boolean}) public horizontal = false
-  @property({type: String})  public contractAddress
-  @property({type: String})  public tokenId
-  @property({type: String})  public width = this.horizontal ? '670px' : '388px'
-  @property({type: String})  public height = this.horizontal ? '250px' : '560px'
-  @property({type: String})  public network = 'mainnet'
-
-  @property({type: String})  private flippedCard = false
-  @property({type: Boolean}) private loading = true
-
-  @property({type: Object})  public asset
-  @property({type: Object})  public traitData = {}
-  @property({type: String})  public account
-
-  private getNetwork() {
-    switch (this.network) {
-      case 'mainnet': return Network.Main
-      case 'rinkeby': return Network.Rinkeby
     }
-  }
-
-  constructor() {
-    super()
-  }
-
-  /**
-   * ConnectedCallback - Invoked when a component is added to the document’s DOM.
-   * Grabs data from the OpenSea SDK and populates data objects to be passed to
-   * child components.
-   */
-  public async connectedCallback() {
-    super.connectedCallback()
-    this.provider =
-      typeof web3 !== 'undefined'
-        ? window.web3.currentProvider
-        : Web3.providers.HttpProvider('https://mainnet.infura.io')
-
-    this.web3 = new Web3(this.provider)
-    this.seaport = new OpenSeaPort(this.provider, {networkName: this.getNetwork()})
-    await this.connectWallet()
-    this.asset = await this.seaport.api.getAsset(this.contractAddress, this.tokenId);
-
-    [this.account] = await this.web3.eth.getAccounts()
-
-    // TODO: Check if account exists
-    if(this.account) {
-        this.asset.isOwnedByAccount = this.asset.owner.address.toLowerCase() === this.account.toLowerCase()
+
+    /**
+     * ConnectedCallback - Invoked when a component is added to the document’s DOM.
+     * Grabs data from the OpenSea SDK and populates data objects to be passed to
+     * child components.
+     */
+    public async connectedCallback() {
+        super.connectedCallback()
+
+        this.seaport = new OpenSeaPort(this.provider, {networkName: this.getNetwork()})
+        await this.connectWallet()
+
+        ;[this.account] = await this.web3.eth.getAccounts()
```

Comment:
this is kinda strange-looking and brittle if someone removes the opening semicolon - can we change it to `this.account = await this.web3.eth.getAccounts()[0]`?